Added CLI link in readme

nirajpandkar opened the pull request.

nirajpandkar commented:
@hemanth I hope the position in the readme is satisfactory.

hemanth commented:
Looks nice, we can do a thing, I shall add you as an npm owner to `git-tips` would you mind renaming `gtips` to `git-tips`?

nirajpandkar commented:
So let me get this clear, you want me to - 
1. Rename gtip -> git-tips in the readme
2. Github-link -> npm link 

That's it right?

nirajpandkar commented:
Or are you talking about adding me as a collaborator in git-tips npm package? And then I should make the corresponding changes and push it to the new package?

nirajpandkar commented:
Ping?

hemanth commented:
> adding me as a collaborator in git-tips npm package

Yes ^

nirajpandkar commented:
Sure. I haven't done this before.. What all would I have to do?
1. Edit `package.json` to change the name.

hemanth commented:
Get me your npm handle.

After I add you to `git-tips` npm, could can `unpublish` `gtips`.

I shall create a `git-tips` repo for node module in here and then you can push the code to it.

What say?

hemanth commented:
Or You can just rename your module and publish it as `git-tips`, naming makes sense, it sounds better than `gtips`?

nirajpandkar commented:
Yes I agree about the naming :+1: I'll just rename my module to git-tips and update the PR.

nirajpandkar commented:
Phew :sweat_smile: After a lot of refactoring, I've managed to change the name to `git-tip`. Deprecated gtip and also changed the link to npm package :+1: 
**Note**: Since the CLI outputs one tip at a time without the arguments I thought it sensible to name it git-tip without the extra 's'.

Updated the PR.

hemanth commented:
Heh heh nice!

Should we add a tools section and add this? 🤔

nirajpandkar commented:
Can't view the emoticon you just posted! And yeah will add a 'Tools' sub-category :+1:

nirajpandkar commented:
How does that look? Nice to see a python implementation. Hope they complete the tool :smile:

hemanth commented on README.md:
```diff
@@ -3,6 +3,10 @@
 
 [English](http://git.io/git-tips) | [中文](https://github.com/521xueweihan/git-tips)
 
+### Tools
```
Sorry, but` __Tools:__` thanks!

nirajpandkar replied:
Done :+1:

nirajpandkar commented:
Anything to make it look good :smile:

nirajpandkar commented:
Ping? @hemanth

hemanth commented:
![tLWWnZB7CLhF6](https://camo.githubusercontent.com/f19aa461ee620601f37f8a0ef2e23c46411a6aa5/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f744c57576e5a4237434c6846362f67697068792e676966>)
